### PR TITLE
Add test tools, fixes #14 

### LINKED ISSUE #14: Add tools for unit-testing

### PR DESCRIPTION
**Pull request by nkbt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,8 @@
   },
   "devDependencies": {
     "babel-eslint": "^3.1.21",
-    "eslint": "^0.24.0"
+    "blue-tape": "^0.1.9",
+    "eslint": "^0.24.0",
+    "tape": "^4.0.0"
```
Comment (nkbt) [inline package.json]: At the moment `tape` is used only to reuse its CLI, until https://github.com/spion/blue-tape/pull/12 is merged

Comment (ericelliott) [inline package.json]: :+1:

Comment (ericelliott) [inline package.json]: Merged. Boom. =)

Comment (nkbt) [inline package.json]: Yep, great.

**Pull request merged**